### ISSUE TITLE
Some books are navigating from right to left

### ISSUE THREAD

**Issue by prsolucoes** [readium/r2-navigator-swift#146]
Hi,

I have a problem with some books that are navigating from right to left and others are normal from left to right.

There is something that change this behaviour inside the epub?

Thanks.

**Comment by prsolucoes** [comment]
Hi,

The is the epub:
https://www.dropbox.com/s/6hu5dmklac5ievl/ebook.epub?dl=1

The OPF:
```
<?xml version="1.0" encoding="utf-8"?>
<package version="3.0" unique-identifier="bookid" prefix="ibooks: http://vocabulary.itunes.apple.com/rdf/ibooks/vocabulary-extensions-1.0/" xmlns="http://www.idpf.org/2007/opf">
  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
    <meta content="Adobe InDesign 15.0.1" name="generator" />
    <meta name="cover" content="CAPA_Obama_1800x1200.jpg" />
    <meta property="ibooks:specified-fonts">true</meta>
    <dc:title>EPUB - Barack Obama</dc:title>
    <dc:date>2020-01-24T17:21:39Z</dc:date>
    <dc:language>ar-SA</dc:language>
    <dc:language>en-US</dc:language>
    <dc:language>pt-BR</dc:language>
    <meta property="dcterms:modified">2020-01-27T15:41:56Z</meta>
    <dc:identifier id="bookid">urn:uuid:30b5b812-89d5-48d4-b131-6fd1b645c8bb</dc:identifier>
    <meta content="0.9.14" name="Sigil version" />
  </metadata>
  <manifest>
    <item id="cover" href="Text/cover.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama" href="Text/EPUB_-_Barack_Obama.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-2" href="Text/EPUB_-_Barack_Obama-2.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-3" href="Text/EPUB_-_Barack_Obama-3.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-4" href="Text/EPUB_-_Barack_Obama-4.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-5" href="Text/EPUB_-_Barack_Obama-5.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-6" href="Text/EPUB_-_Barack_Obama-6.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-7" href="Text/EPUB_-_Barack_Obama-7.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-8" href="Text/EPUB_-_Barack_Obama-8.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-9" href="Text/EPUB_-_Barack_Obama-9.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-10" href="Text/EPUB_-_Barack_Obama-10.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-11" href="Text/EPUB_-_Barack_Obama-11.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-12" href="Text/EPUB_-_Barack_Obama-12.xhtml" media-type="application/xhtml+xml"/>
    <item id="EPUB_-_Barack_Obama-13" href="Text/EPUB_-_Barack_Obama-13.xhtml" media-type="application/xhtml+xml"/>
    <item id="ncx" href="toc.ncx" media-type="application/x-dtbncx+xml"/>
    <item id="toc" href="Text/toc.xhtml" media-type="application/xhtml+xml" properties="nav"/>
    <item id="idGeneratedStyles.css" href="Styles/idGeneratedStyles.css" media-type="text/css"/>
    <item id="HelveticaNeueLTStd-Bd.otf" href="Fonts/HelveticaNeueLTStd-Bd.otf" media-type="application/vnd.ms-opentype"/>
    <item id="HelveticaNeueLTStd-BdIt.otf" href="Fonts/HelveticaNeueLTStd-BdIt.otf" media-type="application/vnd.ms-opentype"/>
    <item id="HelveticaNeueLTStd-Lt.otf" href="Fonts/HelveticaNeueLTStd-Lt.otf" media-type="application/vnd.ms-opentype"/>
    <item id="MinionPro-It.otf" href="Fonts/MinionPro-It.otf" media-type="application/vnd.ms-opentype"/>
    <item id="MinionPro-Regular.OTF" href="Fonts/MinionPro-Regular.OTF" media-type="application/vnd.ms-opentype"/>
    <item id="x1.jpg" href="Images/1.jpg" media-type="image/jpeg"/>
    <item id="x2.jpg" href="Images/2.jpg" media-type="image/jpeg"/>
    <item id="CAPA_Obama_1800x1200.jpg" href="Images/CAPA_Obama_1800x1200.jpg" media-type="image/jpeg" properties="cover-image"/>
    <item id="x3.jpg" href="Images/3.jpg" media-type="image/jpeg"/>
  </manifest>
  <spine toc="ncx">
    <itemref idref="cover" linear="no"/>
    <itemref idref="EPUB_-_Barack_Obama"/>
    <itemref idref="EPUB_-_Barack_Obama-2"/>
    <itemref idref="EPUB_-_Barack_Obama-3"/>
    <itemref idref="EPUB_-_Barack_Obama-4"/>
    <itemref idref="EPUB_-_Barack_Obama-5"/>
    <itemref idref="EPUB_-_Barack_Obama-6"/>
    <itemref idref="EPUB_-_Barack_Obama-7"/>
    <itemref idref="EPUB_-_Barack_Obama-8"/>
    <itemref idref="EPUB_-_Barack_Obama-9"/>
    <itemref idref="EPUB_-_Barack_Obama-10"/>
    <itemref idref="EPUB_-_Barack_Obama-11"/>
    <itemref idref="EPUB_-_Barack_Obama-12"/>
    <itemref idref="EPUB_-_Barack_Obama-13"/>
  </spine>
  <guide>
    <reference type="cover" title="Cover" href="Text/cover.xhtml"/>
  </guide>
</package>
```

**Comment by mickael-menu** [comment]
Okay it was a bug caused when an OPF contains several languages, the first one being RTL (here `ar-SA`), in which case the whole EPUB is considered RTL.

This should be fixed in the latest version (2.0.0-alpha.2), or the `develop` branch.
